packages/edit-post/src/components/options-modal/index.js

```diff
@@ -42,7 +42,9 @@ export function OptionsModal( { isModalActive, closeModal } ) {
 				<EnableTipsOption label={ __( 'Enable Tips' ) } />
 			</Section>
 			<Section title={ __( 'Document Panels' ) }>
-				<EnablePanelOption label={ __( 'Permalink' ) } panelName="post-link" />
+				{ isPermalinkEditable && (
```

Comment:
Even if permalink is not editable we may show the Permalink panel. The conditions to show the permalink panel are complex e.g: we don't show it for new posts etc.
I think the ideal check for this case is ```<PostTypeSupportCheck supportKeys={ [ 'viewable' ] }>``. This way we don't show the option in post types where the permalink panel will never appear, but we show the option for post type where the permalink may appear even if it doe not appear currently e.g. because the post is new.

Reply:
Thanks for your feedback, that's what I did in https://github.com/WordPress/gutenberg/pull/12394/commits/beda76c02447b1e0547720be4c1d695a0df46c5e